internal/inspect/policyset.go

```diff
@@ -42,8 +42,9 @@ func (ps *PolicySet) Inspect(pset *runtimev1.RunnablePolicySet) error {
 	}
 
 	ps.results[namer.PolicyKeyFromFQN(pset.Fqn)] = &responsev1.InspectPoliciesResponse_Result{
-		Actions:   actions,
-		Variables: variables,
+		Actions:         actions,
+		Variables:       variables,
+		StoreIdentifier: pset.Fqn,
```

Comment:
I think this should be the policy key instead of the FQN. Remember to update `ListPolicyIDs` method of the `bundle` driver to also return policy keys instead of FQNs.